### DISCUSSION TITLE
Feature Request: Any plans to add an option to auto sync with Apple health?

### DISCUSSION THREAD

**Discussion by Lemon-ja** [brandonp2412/Flexify#76] [Ideas]
Just installed this and it looks really good and clean, any plan on adding auto sync with Apple health in the future?

**Comment by brandonp2412** [comment]
Interacting with Apple health, Google health or Health Connect would require the inclusion of non-free libraries.

Doing this would go against the spirit of this project and lead to the app being rejected due to the [F-Droid inclusion policy](https://f-droid.org/en/docs/Inclusion_Policy/)
